lang/test/org/partiql/lang/syntax/SqlParserDateTimeTests.kt

```diff
@@ -26,7 +26,12 @@ class SqlParserDateTimeTests : SqlParserTestBase() {
 
     @Test
     @Parameters
```

Comment:
Not exactly sure what you mean here. Do you mean running an individual test in the file is difficult? I am able to run the individual test by clicking the "play icon" next to it.

Reply:
1. Run this test in IntelliJ.
2. In the test runner, it will expand to one test for every case
3. right-click one of the test cases in IntelliJ, notice that it says "nothing here" 
4. Run any test where the test cases are specified with either `@MethodSource` or `@ArgumentsSource` (i.e. a JUnit5 parameterized test)
5. Individual test cases will appear in the test runner as before, however... 
6. Right click on one test case - observe that you can run just that one test case.

Reply:
Ohh, I get it now. Yeah. It indeed is easier to run a single test. Thanks a lot. I will change this in one of the subsequent PRs. Closing this for now.